modules/core/src/main/scala/sangria/renderer/QueryRenderer.scala

```diff
@@ -485,6 +488,13 @@ object QueryRenderer {
   def render(node: AstNode, config: QueryRendererConfig = Pretty, indentLevel: Int = 0): String =
     renderNode(node, config, Indent(config, indentLevel, indentLevel))
 
+  def renderPretty(node: AstNode): String = node match {
+    case _: Value => render(node, PrettyInput)
+    case _ => render(node, Pretty)
+  }
+
+  def renderCompact(node: AstNode): String = render(node, Compact)
```

Comment:
This would be something to backport to the 2.x version before, deprecating the `.renderXXX` methods on the AST. This would be a migration path to smoothly transition to this,

Reply:
Yep. I have [ideas of what to deprecate](https://github.com/performantdata/sangria/discussions/1#discussioncomment-1573890). Just need to see what gets committed.